.github/workflows/wheels.yaml

```diff
@@ -25,32 +26,24 @@ jobs:
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: 3.9
-
-      - uses: s-weigand/setup-conda@v1
-        if: matrix.os == 'windows-latest'
-        with:
-          update-conda: true
-          conda-channels: conda-forge
-          activate-conda: true
-          python-version: 3.9
+          python-version: '3.10'
 
       - uses: actions/checkout@v2
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==1.9.0
+        run: python -m pip install cibuildwheel==2.1.3
 
       - name: Install dependencies and yt
         shell: bash
         env:
-          dependencies: "full"
+          dependencies: "bare"
           LDFLAGS: "-static-libstdc++"
         run: source ./tests/ci_install.sh
 
       - name: Build wheels for CPython
         run: |
           python -m cibuildwheel --output-dir dist
         env:
-          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
+          CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-*"
```

Comment:
This should be the only change necessary to add 3.10 support for wheels.

Reply:
That's what I was going for originally. However I found that
1) updating cibuildwheel to 2.1.3 was needed to use the latest actual Python 3.10.0 release (instead of a beta)
2) we can't update all the way to 2.2.2 for now because it will build wheels for Musllinux and apparently this is currently broken ? see my failed attempt here https://github.com/neutrinoceros/yt/runs/4139228854?check_suite_focus=true
3) we cannot yet install the "full" yt build (including optional dependencies) with Python 3.10 because of h5py. This is why I changed the dependencies target.

Reply:
1. OK, that's fine, but it's should still work with 3.9 as main env
2. `CIBW_SKIP: "*-musllinux-*"` ?
3. The "full" is installed inside the env that runs cibuildwheel, not within the images that build wheels.

Reply:
thanks a lot, I'll try that